src/qml/AboutPage.qml

```diff
@@ -62,12 +84,22 @@ Flickable {
                 { label: qsTr("WLAN MAC"), text: about.wlanMacAddress },
                 { label: qsTr("IMEI"), text: about.imei },
                 { label: qsTr("Serial number"), text: about.serial },
-                { label: qsTr("Total disk space"), text: qsTr("%L1").arg(Math.round(about.totalDiskSpace() / 1e7)/100) + " GB" },
-                { label: qsTr("Available disk space"), text: qsTr("%L1").arg(Math.round(about.availableDiskSpace() / 1e7)/100)
-                            + " GB (" + (100.0 * about.availableDiskSpace() / about.totalDiskSpace()).toFixed(0) + "%)" },
-                { label: qsTr("Display size"), text: Dims.w(100) + qsTr("W") + " x " + Dims.h(100) + qsTr("H") },
+                { label: qsTr("Total disk space"), text: qsTr("%L1 GB").arg(Math.round(about.totalDiskSpace() / 1e7)/100) },
+                { label: qsTr("Available disk space"), text: qsTr("%L1 GB (%L2 %)").
+                    arg(Math.round(about.availableDiskSpace() / 1e7)/100).
+                    arg((100.0 * about.availableDiskSpace() / about.totalDiskSpace()).toFixed(0)) },
+                { label: qsTr("Display size"), text: qsTr("%L1W x %L2H").arg(Dims.w(100)).arg(Dims.h(100)) },
                 { label: qsTr("Kernel version"), text: kernelVersion },
-                { label: qsTr("Qt version"), text: qtVersion }
+                { label: qsTr("Qt version"), text: qtVersion },
+                { label: qsTr("uptime"), text: qsTr("%L1 days %L2").
```

Comment:
It's a bit awkward to write "0 days" here. How about having two translatable strings: one that contains %L1 days %L2 and the other one containing only %L2 and using one or the other depending on whether days is null or not ?
Extracting this logic in a helper function like you've mostly done so far sounds like a right thing

Reply:
I was noting that too. On the one hand, 0 days is an information to and avoid the question if they are counted at all our the counter wraps daily. Otho, it looks cleaner with just %L2.
No strong opinion here, trade offs everywhere :shrug:

Reply:
One could also note that the "1 days" is grammatically incorrect and it should be "1 day" in English. To allow for translation of plurals, the format string "%n" is used, but the "%L" form allows localization of the number in case a watch is running for three years and we need to report an uptime of "1,002 days".  Which one to choose?

Reply:
I changed it to use "%n" format string to allow translation.  I think it's useful to know if it's 0 days.